### ISSUE TITLE
Removed triggers fail to work

### ISSUE THREAD

**Issue by ZachHaber** [Kurohyou/Roll20-Snippets#27]
https://github.com/Kurohyou/Roll20-Snippets/blob/ce83d8877aa383ad0f10d2c29988c5e2e5729b64/K_Scaffold/Template%20v2/scaffold/attribute_proxy.js#L8-L13

In this bit, the removed eventName ends up being `removed:repeating_section`, which makes it search for `fieldset_removed:repeating_section`, so along with removing `clicked:`, `removed:` also needs to be removed, and then things can properly listen to repeater items being removed.

```
      let typePrefix = 'attr_';
      if (eventName.startsWith('clicked:')) {
        typePrefix = 'act_';
        eventName = eventName.replace('clicked:', '');
      } else if (event.removedInfo) {
        typePrefix = 'fieldset_';
        eventName = eventName.replace('remove:', '');
      }
      return casc[`${typePrefix}${eventName}`];
```

**Comment by Kurohyou** [comment]
Thank you Zach!

**Comment by Kurohyou** [comment]
Should be resolved by #30
